### PR TITLE
[16.0][ADD] fleet_vehicle_usage: New module

### PR DESCRIPTION
**Pull request by victoralmau**
New module

Please @pedrobaeza and @carolinafernandez-tecnativa can you review it?

@Tecnativa TT47818

### CODE REVIEW

**Review by marcelsavegnago — APPROVED** [review]
LGTM

File: fleet_vehicle_usage/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 César Fernández Domínguez
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Fleet Vehicle Usage",
+    "version": "16.0.1.0.0",
+    "category": "Human Resources/Fleet",
+    "website": "https://github.com/OCA/fleet",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
```
Comment (pedrobaeza) [inline fleet_vehicle_usage/__manifest__.py]:
```suggestion
    "author": "César Fernández, Tecnativa, Odoo Community Association (OCA)",
```

File: fleet_vehicle_usage/data/ir_sequence_data.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="sequence_fleet_vehicle_usage" model="ir.sequence">
+        <field name="name">Fleet Vehicle Usage</field>
+        <field name="code">fleet.vehicle.usage</field>
+        <field name="prefix">FU/</field>
+        <field name="padding">5</field>
+        <field name="number_next">1</field>
+        <field name="number_increment">1</field>
```
Comment (pedrobaeza) [inline fleet_vehicle_usage/data/ir_sequence_data.xml:9]: Add explicitely `company_id`=False for being multi-company by default.

File: fleet_vehicle_usage/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+This module adds record the use of fleet vehicles by employees.
```
Comment (pedrobaeza) [inline fleet_vehicle_usage/readme/DESCRIPTION.rst]:
```suggestion
This module allow to record usages of fleet vehicles by employees, with their dates, states and comments.
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by carolinafernandez-tecnativa — APPROVED** [review]
Funcional review, LGTM :) Thanks!!

File: fleet_vehicle_usage/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1,2 @@
+This module allow to record usages of fleet vehicles by employees, with their dates,
```
Comment (pedrobaeza) [inline fleet_vehicle_usage/readme/DESCRIPTION.rst]: Sorry, my bad:

```suggestion
This module allows to record usages of fleet vehicles by employees, with their dates,
```

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**